scripts/genomes/parse_genome_assemblies.py

```diff
@@ -0,0 +1,208 @@
+"""Parse genome assemblies from NCBI by their level of completion.
+
+This script fetches current and historical Assembly Reports from NCBI, and
+extracts metadata on assemblies that are relevant for Single Cell Portal.
+
+Only assemblies that are chromosome-level or above are included.
+
+For definitions of assembly-related terms and other domain background, see:
+https://www.ncbi.nlm.nih.gov/assembly/help/#glossary
+"""
+
+import argparse
+from functools import cmp_to_key
+import os
+import urllib.request as request
+
+parser = argparse.ArgumentParser(
+    description=__doc__, # Use docstring at top of file for --help summary
+    formatter_class=argparse.RawDescriptionHelpFormatter)
+parser.add_argument('--output_dir',
+                    help='Directory to sent output data to.  Default: output/',
```

Comment:
typo? sent -> send

Reply:
Good catch; this is fixed in my current topic branch, [`ew-genome-annotations`](https://github.com/broadinstitute/single_cell_portal/tree/ew-genome-annotations).